pipeline/metadata/flatten_hyperquack.py

```diff
@@ -73,6 +76,8 @@ def process_hyperquack(self, filename: str, scan: Any,
     Yields:
       Rows
     """
+    Metrics.counter(METRIC_NAMESPACE, NUM_LINES_METRIC_NAME).inc()
```

Comment:
We would never be outputting both hyperquack and satellite measurements at the same time so these don't need to be labeled as hyperquack. I actually want to reuse the same metric for satellite, but I didn't do it here because the satellite flattening is more complex to handle correctly.

Changed to MEASUREMENT_LINES